src/Arcus.EventGrid.Tests.InMemoryApi/Arcus.EventGrid.Tests.InMemoryApi.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp3.1</TargetFrameworks>
   </PropertyGroup>
```

Comment:
Changed this, but once the obsolete attributes are removed, we can actually remove this in-memory project.